docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -173,6 +173,60 @@ default:: `true`
 
 This parameter controls whether the component enables monitoring on the `CephCluster` resource and sets up Ceph alert rules.
 
+=== `storageClassDeviceSets`[[ceph_cluster_storageclass]]
+
+[horizontal]
+type:: dict
+keys:: Names of `storageClassDeviceSet` resources
+values:: Spec of `storageClassDeviceSet` resources
+
+The component creates adds a storage class device sets to the cluster object for every entry.
+The key is used as the name of the storage class device set and the value is used as the specification with the `volumeClaimTemplates` key coverted from a dictionary to a list.
+
+This means the following example configuration:
+
+[source,yaml]
+----
+storageClassDeviceSets:
+  foo:
+    count: 3
+    volumeClaimTemplates:
+      default:
+        spec:
+          storageClassName: localblock
+          volumeMode: Block
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 100Gi
+----
+
+Would be converted to the following specification
+
+[source,yaml]
+----
+storageClassDeviceSets:
+  - name: foo
+    count: 3
+    volumeClaimTemplates:
+      - spec:
+          storageClassName: localblock
+          volumeMode: Block
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 100Gi
+----
+
+See https://rook.io/docs/rook/v1.6/ceph-cluster-crd.html#storage-class-device-sets[Storage Class Device Sets] for a more detailed explanation on how to configure them.
+
+NOTE: The default storage class device set is called `cluster` and can also be configured through this parameter.
+
+IMPORTANT: If you set `cephClusterSpec.storage.storageClassDeviceSets`, making changes to this parameter isn't supported.
```

Comment:
As far as I can see the implementation simply appends device sets defined in `cephClusterSpec.storage.storageClassDeviceSets` to the device sets rendered from this parameter, because we ensure that the contents of parameter `cephClusterSpec` are merged with the existing config in line 239 in `cephcluster.libsonnet`.